app/workers/data_request_worker.rb

```diff
@@ -10,8 +10,7 @@ class DataRequestWorker
   attr_accessor :path
 
   def perform(request_id)
-    request = DataRequest.find(request_id)
-    # return unless request.status == DataRequest::PENDING
+    request = DataRequest.where(id: request_id).first
```

Comment:
If you switch to this style, it might return nil, so you'll want to handle that case with a return unless request.present? Or something

Reply:
good catch!